src/css/rating_pop_up.css

```diff
@@ -0,0 +1,164 @@
+.rating-pop-up-backdrop {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 999;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(4, 4, 4, 0.4);
+  opacity: 0;
+  visibility: hidden;
+  pointer-events: none;
+  display: flex;
+  align-items: center;
+  padding: 0 20px;
+  overflow: auto;
+}
+
+.rating-pop-up-backdrop.is-open {
+  opacity: 1;
+  visibility: visible;
+  pointer-events: initial;
+}
+
+.rating-pop-up-container {
+  position: relative;
+  margin: auto;
+  width: 100%;
+  max-width: 335px;
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  align-items: center;
+  padding: 40px 20px;
+  background-color: var(--dark);
+  border-radius: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .rating-pop-up-container {
+    max-width: 430px;
+    padding: 40px;
+  }
+}
+
+.rating-pop-up-close-btn {
+  background: none;
+  border: none;
+  padding: 0;
+  position: absolute;
+  top: 14px;
+  right: 14px;
+  cursor: pointer;
+}
+
+.rating-pop-up-close-icon {
+  stroke: var(--light);
+  transition: stroke var(--transition);
+}
+
+.rating-pop-up-content {
+  align-self: flex-start;
+}
+
+.rating-data-parameter-name {
+  color: var(--light-40);
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 1.5;
+  margin-bottom: 8px;
+}
+
+.rating-pop-up-content .rating {
+  align-items: center;
+}
+
+.rating-pop-up-content .rating .rating-star-container {
+  width: 24px;
+  height: 24px;
+  cursor: pointer;
+}
+.rating-pop-up-content .rating .rating-star-container .rating-star {
+  width: 20px;
+  height: 20px;
+}
+.form-rating {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding-top: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .form-rating {
+    padding-top: 32px;
+  }
+}
+
+.rating-form-input {
+  width: 100%;
+  height: 42px;
+  padding: 12px 14px;
+  border-radius: 30px;
+  border: 1px solid var(--light);
+  background-color: var(--dark);
+  transition: all var(--transition);
```

Comment:
Do not apply transition to all properties, but instead list them in the `transition-property: ...`

Reply:
Done